ontology/yaml/resources/HVAC/entity_types/PMP.yaml

```diff
@@ -43,6 +43,77 @@ PMP_SS_VSC_WDPM:
   - VSC
   - WDPM
 
+PMP_SS_WDT:
+  description: "Variable speed pump with temperature reading between return/supply water"
+  is_canonical: true
+  implements:
+  - PMP
+  - SS
+  - WDT
```

Comment:
Move to system.

Reply:
This was wrongly defined on pump, this has been moved to HWS and renamed as HWS_SS_WDT